x/lockup/keeper/lock_test.go

```diff
@@ -369,47 +369,43 @@ func (suite *KeeperTestSuite) TestAddTokensToLock() {
 			expectAddTokensToLockSuccess: true,
 		},
 		{
-			name:                         "not the owner of the lock",
-			tokenToAdd:                   initialLockCoin,
-			duration:                     time.Second,
-			lockingAddress:               addr2,
-			expectAddTokensToLockSuccess: false,
+			name:           "not the owner of the lock",
+			tokenToAdd:     initialLockCoin,
+			duration:       time.Second,
+			lockingAddress: addr2,
 		},
 		{
-			name:                         "lock with matching duration not existing",
-			tokenToAdd:                   initialLockCoin,
-			duration:                     time.Second * 2,
-			lockingAddress:               addr1,
-			expectAddTokensToLockSuccess: false,
+			name:           "lock with matching duration not existing",
+			tokenToAdd:     initialLockCoin,
+			duration:       time.Second * 2,
+			lockingAddress: addr1,
 		},
 		{
-			name:                         "lock invalid tokens",
-			tokenToAdd:                   sdk.NewCoin("unknown", sdk.NewInt(10)),
-			duration:                     time.Second,
-			lockingAddress:               addr1,
-			expectAddTokensToLockSuccess: false,
+			name:           "lock invalid tokens",
+			tokenToAdd:     sdk.NewCoin("unknown", sdk.NewInt(10)),
+			duration:       time.Second,
+			lockingAddress: addr1,
 		},
 		{
-			name:                         "token to add exceeds balance",
-			tokenToAdd:                   sdk.NewCoin("stake", sdk.NewInt(20)),
-			duration:                     time.Second,
-			lockingAddress:               addr1,
-			expectAddTokensToLockSuccess: false,
-			expectedError:                true,
+			name:           "token to add exceeds balance",
+			tokenToAdd:     sdk.NewCoin("stake", sdk.NewInt(20)),
+			duration:       time.Second,
+			lockingAddress: addr1,
+			expectedError:  true,
 		},
 	}
 
 	for _, tc := range testCases {
 		suite.SetupTest()
 		// lock with balance
 		suite.FundAcc(addr1, sdk.Coins{initialLockCoin})
-		lock, err := suite.App.LockupKeeper.CreateLock(suite.Ctx, addr1, sdk.Coins{initialLockCoin}, time.Second)
+		originalLock, err := suite.App.LockupKeeper.CreateLock(suite.Ctx, addr1, sdk.Coins{initialLockCoin}, time.Second)
```

Comment:
Out of curiosity, what's the point of clarifying that this is the "original" lock? Doesn't `AddToExistingLock` mutate it and make it non-original?

Reply:
Good question! This is not a pointer reference to the lock so `AddToExistingLock` would not make this non-original

Reply:
Ah I see, thanks!